cartoframes/contrib/vector.py

```diff
@@ -378,6 +428,12 @@ def vmap(layers,
           properties, or a list of floats in the following order: [west,
           south, east, north]. If not provided the bounds will be automatically
           calculated to fit all features.
+        viewport (dict): Configure where and how map will be centered.
+            - lng (float): Longitude to center the map on.
+            - lat (float): Latitude to center the map on.
+            - zoom (float): Zoom level.
+            - bearing (float): Bearing of the map, in degrees. 0 means north is 'up'.
```

Comment:
A description like from Mapbox GL's docs would be helpful:
> A bearing, or heading, is the direction you’re facing, measured clockwise as an angle from true north on a compass (north is 0°, east is 90°, south is 180°, and west is 270°).

https://docs.mapbox.com/help/glossary/bearing/

Pitch from mapbox gl `camera` docs:
> Pitch: The angle towards the horizon measured in degrees, with a range between 0 and 60 degrees. Zero degrees results in a two-dimensional map, as if your line of sight forms a perpendicular angle with the earth's surface.

Reply:
Will use those :+1: